cmd/zetae2e/populate_addresses.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/gagliardetto/solana-go"
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/node/e2e/config"
+)
+
+func NewPopulateAddressesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "populate-addresses [config-file] ",
+		Short: "Derive addresses from the configured private keys and populate in the config file",
+		RunE:  runPopulateAddresses,
+		Args:  cobra.ExactArgs(1),
+	}
+	return cmd
+}
+
+func runPopulateAddresses(_ *cobra.Command, args []string) error {
+	conf, err := config.ReadConfig(args[0], false)
```

Comment:
Can we precise why we don't validate the config in this case?

Reply:
Updated